.github/workflows/docker_build.yml

```diff
@@ -1,62 +1,59 @@
 name: Build and Push Docker Images on release/
 on:
-    workflow_call:
-        inputs:
-            image_name:
-                required: true
-                type: string
-                description: name of the image to build
-            tag_name:
-                required: true
-                type: string
-                description: name of the tag to build
-            app_name:
-                required: true
-                type: string
-                description: name to tag the image with
-            app_directory:
-                required: false
-                type: string
-                description: directory that is the root context, defaults to docker/app_name
-            dockerfile_name:
-                required: false
-                default: "Dockerfile"
-                type: string
-                description: Name of the Dockerfile, defaults to 'Dockerfile'
```

Comment:
Would be useful to keep this property, as it is not required (and defaults to "Dockerfile") but there may be multiple projects to build with docker in a single context, or not every app may have a conforming file named "Dockerfile".

Reply:
I don't think it would be useful. Specifying the context should be enough. And we should always name our dockerfiles `Dockerfile` and not something else.